build/protoc-go/Dockerfile

```diff
@@ -3,6 +3,10 @@ FROM golang:1.12
 RUN apt-get update && \
     apt-get install -y protobuf-compiler && \
     go get -u github.com/golang/protobuf/protoc-gen-go && \
+    go get -u github.com/gogo/protobuf/protoc-gen-gofast && \
+    go get -u github.com/gogo/protobuf/protoc-gen-gogofast && \
+    go get -u github.com/gogo/protobuf/protoc-gen-gogofaster && \
```

Comment:
I'm curious why you decided not to use protoc-gen-gogogofastest though.

Reply:
Too fast

Reply:
I didn't want to overheat my laptop

Reply:
Jordan. Do you want to tackle issue #560 at the same time in this PR and upgrade to 3.9.0?

Reply:
Sure sounds good. I have another small change to make to this one anyways.